Casks/ihcask.rb

```diff
@@ -0,0 +1,11 @@
+cask "ihcask" do
+    version "1.7.0"
+    sha256 "41feea152b3dcff8fb729106b195e4dc7632cda669ef7054045f72c595825242"
+      
+    url "https://github.com/rancher-sandbox/rancher-desktop/releases/download/v#{version}/Rancher.Desktop-#{version}.aarch64.dmg"
```

Comment:
We currently have a mix of M1 (`aarch64`) and x64 (`amd64`, probably) laptops in use in engineering, so this will need to be a little more clever.